api/questions/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
You can also register questions model to admin panel

Reply:
Thanks for the reminder, had forgotten about that